### PR TITLE
feat: add eslint with airbnb extended

### PR DESCRIPTION
**Pull request by phetw**
Referencing to issue #4 , the pull request consist of 2 commits. The first one is installing packages and configurations and the second one is fixing lint errors.

### CODE REVIEW

**Comment by shubhamzanwar** [comment]
Nice PR. just one suggestion. Can you use .js files instead of .jsx?
easier to review and that is the accepted standard now-a-days. In case an eslint rule breaks because of that, feel free to disabble that particular rule :)

**Comment by phetw** [comment]
Hi, @shubhamzanwar by default `eslint-plugin-react` restricts jsx syntax only in .jsx file. There is an option to allow it though through `.eslintrc` see [this](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-filename-extension.md).

Let me know which one do you prefer.

**Comment by phetw** [comment]
@shubhamzanwar Done : )

File: package.json [hunk 1]
```diff
@@ -25,14 +26,24 @@
...
     "webpack": "^4.39.3",
     "webpack-cli": "^3.3.7",
-    "webpack-dev-server": "^3.8.0",
-    "react": "^16.8.0"
+    "webpack-dev-server": "^3.8.0"
   },
   "peerDependencies": {
     "react": "^16.8.0"
+  },
+  "dependencies": {
```
Comment (shubhamzanwar) [inline package.json]: Can we move this to dev dependencies please? Don't think we need it as a dependency once the bundle is formed. That way, the library size will remain small :)

**Review by shubhamzanwar — APPROVED** [review]

**Comment by shubhamzanwar** [comment]
closes #4

**Pull request merged**